src/transformers/models/encoder_decoder/configuration_encoder_decoder.py

```diff
@@ -26,8 +26,9 @@
 class EncoderDecoderConfig(PretrainedConfig):
     r"""
     :class:`~transformers.EncoderDecoderConfig` is the configuration class to store the configuration of a
-    :class:`~transformers.EncoderDecoderModel`. It is used to instantiate an Encoder Decoder model according to the
-    specified arguments, defining the encoder and decoder configs.
+    :class:`~transformers.EncoderDecoderModel` or a :class:`~transformers.FlaxEncoderDecoderModel`. It is used to
```

Comment:
Actually I think we can leave this one here as it was :-) We usually only use `PyTorch`s classes as a reference in tokenizer and configuration files. We kinda assume that every functionality that exists has to exist at least in PyTorch. Sorry for only noting this now!